contributing.md

```diff
@@ -0,0 +1,122 @@
+# Contributing Quickstart
+
+## Purpose
+
+This guide will help you quickly set up a development environment where you can contribute code to Handy. This guide will be especially useful for newcomers to Rails development.
+
+Specifically, this guide will show you how to use VirtualBox and Vagrant to set up your dev environment with a virtual machine (VM). If you feel comfortable with Rails and don't want to use a VM, this guide isn't for you.
+
+## Prerequisites
+
+We'll use VirtualBox and Vagrant to prepare our environment.
+
+Go [here](https://www.virtualbox.org/) to get VirtualBox for your machine.
+
+Then, go [here](https://www.vagrantup.com/) to get Vagrant.
+
+VirtualBox is a VM provider. It does the actual work of hosting a fake computer (the VM) in your computer (the host). Vagrant is a tool that works on top of VirtualBox. It makes sharing and configuring development environments easy.
+
+Once you've installed the tools, I recommend going through Vagrant's [_Getting Started_](http://docs.vagrantup.com/v2/getting-started/index.html) guide so you understand what's going on and you're confident your installations were successful. Or don't. I'm not the boss of you.
+
+## Get the Quickstart Box
+
+I created a "base box" configured specially for Handy development. It's an Ubuntu (Linux) image with all the dependencies you'll need to start the development server*. You can download it from my Google Drive [here](https://drive.google.com/file/d/0B4TO8N-e7-6YbDhBZXROY19xY0k/view?usp=sharing). 
+
+Once you've downloaded handy-quickstart.box, let Vagrant know it exists like so:
+`vagrant box add handy-quickstart handy-quickstart.box`
```

Comment:
If we distribute the base box via [Atlas](https://atlas.hashicorp.com/), there's no need to download it and add it to Vagrant manually. We can reference it in the Vagrantfile and Vagrant will know how to get it:

```ruby
Vagrant.configure(2) do |config|
  config.vm.box = 'pittcsc/handy-box'
  # ...
end
```

Just another option.